library/src/main/java/com/github/paolorotolo/appintro/AppIntroBase.java

```diff
@@ -357,6 +368,13 @@ protected void setButtonState(View button, boolean show) {
     }
 
     /**
+     * Set typeface to only title
+     * @param tf_string URL of font file location in the Assets folder.
+     */
+    protected void setTitleTypeface(String tf_string){
+
```

Comment:
Nothing. I have ported that thing to AppIntroFragment.newInstance method. It is a useless method. I shall remove that. 
My bad.